src/lib.rs

```diff
@@ -118,7 +118,12 @@ pub struct LCh {
 // those are different values than those provided by CIE standard.
 pub(crate) const KAPPA: f32 = 24389.0 / 27.0;
 pub(crate) const EPSILON: f32 = 216.0 / 24389.0;
-pub(crate) const CBRT_EPSILON: f32 = 0.20689655172413796;
+pub(crate) const CBRT_EPSILON: f32 = 6.0 / 29.0;
+
+// S₀ and E₀ thresholds used in sRGB gamma.  The latter is scaled for encoded
+// value in the range 0..255.
+const S_0: f32 = 0.003130668442500564;
+const E_0_255: f32 = 3294.6 * S_0;
```

Comment:
Could you `pub(crate)` both of these, then use them in [`simd/labs_to_rgbs.rs#197`](https://github.com/TooManyBees/lab/blob/95f8ff24311139616ea9517634adea7ff7810f18/src/simd/labs_to_rgbs.rs#L197) and [`simd/rgbs_to_labs.rs#180`](https://github.com/TooManyBees/lab/blob/95f8ff24311139616ea9517634adea7ff7810f18/src/simd/rgbs_to_labs.rs#L180)

Reply:
why am I asking you to do this. i already made the changes on my end to test them.